root/layout/index.js

```diff
@@ -135,9 +143,11 @@ const Layout = ({$c, ...props}) => (
         </div>}
 
       {!!($c.stash.new_edit_notes &&
-          $c.stash.new_edit_notes_mtime >
-          getRequestCookie($c.req, 'new_edit_notes_dismissed_mtime', 0) &&
-          ($c.user.is_limited ||
+          ($c.stash.new_edit_notes_mtime ?? Infinity) >
```

Comment:
Doesn't this cause a risk that if new_edit_notes_mtime fails to be set for some reason, we will always show the banner? (maybe I'm missing something and even if that happens it probably isn't a big deal, but).

Reply:
Yeah, but if `$c.stash.new_edit_notes` is set while `$c.stash.new_edit_notes_mtime` isn't (which would be a bug), I figured the safer default is to show the banner rather than hide it. (If dismissing it doesn't work, going to `/edit/notes-received` probably would, but it really shouldn't happen to begin with and this was mainly to shut up Flow.)